packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -91,11 +91,8 @@ const gnosisSafeContract = new GnosisSafe(GNOSIS_SAFE_ADDRESS);
 const provider = ProviderFactory.getProvider();
 
 const cancelSellOrder = async (nftAddress, id) => {
-  const action = marketplaceContract.cancelOrder(
-    nftAddress,
-    `${id}`,
-    gasParams
```

Comment:
I really like that this (`gasParams`) is gone from this script now.

Reply:
Yes, me too. Now we can handle gas just inside of the `Action`.

Reply:
Yep, cool! Resolving this